dts/bindings/sensor/avago,apds9253.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright(c) 2018, Phytec Messtechnik GmbH
+# SPDX - License - Identifier : Apache - 2.0
+
+description: APDS9253 ambient light, RGB
+
+compatible: "avago,apds9253"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: Interrupt pin.
+
+      The interrupt pin of APDS9253 is open-drain, active low.
+      If connected directly the MCU pin should be configured
+      as pull-up, active low.
+
+  rate:
+    type: int
```

Comment:
Consider type `enum` for the types that require specific values instead of referencing the header file.